wiki/Featured_Artists/en.md

```diff
@@ -3,23 +3,23 @@ tags:
   - FA
 ---
 
-# Featured artists
+# Featured Artists
 
-**Featured artists** are music artists that have worked with osu! to make a selection of their tracks available for use in [beatmaps](/wiki/Beatmap) and background music in the [osu! World Cup](/wiki/Tournaments#osu!-world-cup) live streams.
+**Featured Artists** are music artists that have worked with osu! to make a selection of their tracks available for use in [beatmaps](/wiki/Beatmap) and background music in the [osu! World Cup](/wiki/Tournaments#osu!-world-cup) live streams.
```

Comment:
I'd say that if you capitalize FA, you need to distinguish between Feature Artists, the program, and feature artists, a set of collectives and composers featured in osu! (this line currently means the latter)

Reply:
```suggestion
**Featured artists** are music artists that have worked with osu! to make a selection of their tracks available for use in [beatmaps](/wiki/Beatmap) and background music in the [osu! World Cup](/wiki/Tournaments#osu!-world-cup) live streams.
```

Reply:
Don't think so? This reads fine to me.